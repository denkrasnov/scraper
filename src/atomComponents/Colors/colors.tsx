import { Colors } from "./types";

const colors: Colors = {
  BACKGROUND: "#f9fbfc",
  SHADOW_COLOR: "rgba(8, 31, 44, 0.12)",
  BUTTON_PRIMARY: "#224dff",
  BUTTON_HOVER: "#0b2fc4",
  BORDER: "#c8e1ff",
  BORDER_DARK: "rgba(18, 67, 138, 0.7)",
  BORDER_GRAY: "#d3d3d3",
  SUCCESS: "#45c42d",
  ERROR: "#ff5267",
  BODY_TEXT: "#090f29",
  MAIN: "#f7faff",
  MAIN_DARK: "rgba(133, 204, 255, 0.15)"
};

export default colors;
