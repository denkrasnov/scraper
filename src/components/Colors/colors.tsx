import { Colors } from "./types";

const colors: Colors = {
  BACKGROUND: "var(--backgound)",
  SHADOW_COLOR: "var(--shadow-color)",
  BUTTON_PRIMARY: "var(--button-primary)",
  BUTTON_SECONDARY: "var(--button-secondary)",
  BUTTON_HOVER: "var(--button-hover)",
  BORDER: "var(--border)",
  BORDER_DARK: "var(--border-dark)",
  BORDER_GRAY: "var(--border-gray)",
  SUCCESS: "var(--success)",
  ERROR: "var(--error)",
  BODY_TEXT: "var(--body-text)",
  MAIN: "var(--main)",
  MAIN_DARK: "var(--main-dark)",
  MAIN_RED: "var(--main-red)",
  WHITE: "var(--white)"
};

export default colors;
