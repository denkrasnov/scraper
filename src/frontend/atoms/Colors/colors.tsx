import { Colors } from "./types";

const colors: Colors = {
  BACKGROUND: "var(--background)",
  SHADOW_COLOR: "var(--shadow-color)",
  BUTTON: "var(--button-primary)",
  BUTTON_TRANSPARENT: "var(--button-primary)",
  BUTTON_SECONDARY: "var(--button-secondary)",
  BUTTON_HOVER: "var(--button-hover)",
  BORDER: "var(--border)",
  BORDER_DARK: "var(--border-dark)",
  BORDER_GRAY: "var(--border-gray)",
  SUCCESS: "var(--success)",
  ERROR: "var(--error)",
  BODY_TEXT: "var(--body-text)",
  MAIN: "var(--main)",
  MAIN_DARK: "var(--main-dark)",
  MAIN_RED: "var(--main-red)",
  WHITE: "var(--white)"
};

export default colors;
