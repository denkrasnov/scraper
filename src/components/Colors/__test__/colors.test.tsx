import colors from "..";

describe("colors", () => {
  it("renders", () => {
    expect(colors).toMatchSnapshot();
  });
});
