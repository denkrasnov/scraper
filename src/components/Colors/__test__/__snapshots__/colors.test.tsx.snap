// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`colors renders 1`] = `
Object {
  "BACKGROUND": "var(--backgound)",
  "BODY_TEXT": "var(--body-text)",
  "BORDER": "var(--border)",
  "BORDER_DARK": "var(--border-dark)",
  "BORDER_GRAY": "var(--border-gray)",
  "BUTTON_HOVER": "var(--button-hover)",
  "BUTTON_PRIMARY": "var(--button-primary)",
  "BUTTON_SECONDARY": "var(--button-secondary)",
  "ERROR": "var(--error)",
  "MAIN": "var(--main)",
  "MAIN_DARK": "var(--main-dark)",
  "SHADOW_COLOR": "var(--shadow-color)",
  "SUCCESS": "var(--success)",
  "WHITE": "var(--white)",
}
`;
